### DISCUSSION TITLE
[Projects beta proposal] Move issue to "In Progress" when linked to a PR

### DISCUSSION THREAD

**Discussion by TheFabbiusCorp** [community/community#10442] [Projects and Issues]
With this suggestion, I'd like to propose another default workflow until the upcoming 'editing of workflows' feature is released. IMO, one thing that's missing is the automatic move to the "In Progress" column (if available) when the issue is actually worked on, that is, in most cases, when a PR is linked to that issue. Then, either the PR is closed/merged, so the issue is moved to "Done", or the issue is unlinked to the PR, so it will be moved back to the column where it came from (No Status or To Do).

**Comment by agjohnson** [comment]
I found myself wanting this behavior, but for linked pull request review requests. I was surprised that the current workflow rules do not update the issue status to `Status: Review requested` when review is requested on the linked pull request.
